torch/_tensor_docs.py

```diff
@@ -5937,13 +5937,29 @@ def callable(a, b) -> number
 add_docstr_all(
     "uniform_",
     r"""
-uniform_(from=0, to=1) -> Tensor
+uniform_(from, to, generator) -> Tensor
```

Comment:
I would change to: `uniform_(from=0, to=1, *, generator=None) -> Tensor`
- Keep the default values?
- We should indicate generator is a kwarg-only argument